graphql/subcourse/fields.ts

```diff
@@ -148,14 +148,16 @@ export class ExtendedFieldsSubcourseResolver {
         @Arg('search') search: string,
         @Arg('courseStates', () => [String], { nullable: true }) courseStates: course_coursestate_enum[],
         @Arg('take', () => GraphQLInt) take: number,
-        @Arg('skip', () => GraphQLInt, { nullable: true }) skip: number = 0
+        @Arg('skip', () => GraphQLInt, { nullable: true }) skip: number = 0,
+        @Arg('orderDesc', { nullable: true }) orderDesc: boolean
```

Comment:
We could potentially just expose the Prisma OrderBy type here (there should be a generated GraphQL Type for it) - Then we are flexible towards future requirements

Reply:
Also maybe rather orderByLastUpdate or something like that? orderDesc is a bit generic

Reply:
Or maybe orderBy: "last-update", then we can add further sorting variants in the future?

Reply:
Went for the third option now, sounds like a good idea.